manifests/k/Kindle/Create/1.63.0.0/Kindle.Create.locale.en-us.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Kindle.Create
+PackageVersion: 1.63.0.0
+PackageLocale: en-us
+Publisher: Amazon
+PackageName: Kindle Create
+License: MIT License

```

```suggestion
License: Proprietary
```
Is this really MIT License? Proprietary seems the right one